doc/source/admin/galaxy_options.rst

```diff
@@ -4823,13 +4801,16 @@
 :Default: ``plugins/welcome_page/new_user/static/topics/``
 :Type: str
 
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+~~~~~~~~~~~~~~~~~~~~~
 ``vault_config_file``
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~~~
 
 :Description:
-    Vault configuration
-    The value of this option will be resolved with respect to
-    <config_dir>.
+    Vault config file. The value of this option will be resolved with
+    respect to <config_dir>.
 :Default: ``vault_conf.yml``
 :Type: str
+
```

Comment:
If someone knows where those 3 newlines are coming from I'd be so happy ... 🙏

Reply:
It looks like they were actually removed (manually?) in https://github.com/galaxyproject/galaxy/pull/12940/files#diff-63e15fe9fdbbfc026e332aff2ecd8f44c56a7af1ba315df61e86bce8fd4905e8 , so nothing to worry about?

Reply:
I had these show up in another config rebuild I did recently, too -- definitely nothing to worry about.

Reply:
Alright, then I'll drop them manually

Reply:
Do they mess anything up?  I'd just leave them as whatever gets automatically built w/ make config-rebuild.